src/ring_theory/polynomial/eisenstein.lean

```diff
@@ -156,6 +156,15 @@ end
 
 end comm_ring
 
+lemma _root_.scale_roots.is_weakly_eisenstein_at {R : Type*} [comm_ring R] (p : R[X]) {x : R}
```

Comment:
There is no `R` already declared here? You can see the list of variables using `#where` before this line.